config/examples/AnyCubic/i3 Mega/Trigorilla AVR/Configuration.h

```diff
@@ -1736,7 +1745,7 @@
 #define Y_BED_SIZE 210
 
 // Travel limits (linear=mm, rotational=°) after homing, corresponding to endstop positions.
-#define X_MIN_POS -5
+#define X_MIN_POS 0
```

Comment:
@sobil — Is it true that when X is homed the nozzle is over the bed in a usable location?

Reply:
Yes, that is correct

Reply:
Hey @thinkyhead 

I've just realised that this shouldn't be changed. as some printers appear to be shipped with a slightly smaller bed than: 
![image](https://user-images.githubusercontent.com/1817358/206818564-04332889-5591-4e5f-91f9-3e90f60474e0.png)